intl/crowdin.yaml

```diff
@@ -1,20 +1,21 @@
+_secret_ _secret_ _secret_[_secret_
+_secret_ _secret_ _secret_ _secret_ _secret_{_secret_
+_secret_ _secret_ _secret_ _secret_ _secret_ _secret_ _secret_"_secret_s_secret_o_secret_u_secret_r_secret_c_secret_e_secret_"_secret_:_secret_ _secret_"_secret_/_secret_m_secret_s_secret_g_secret___secret_h_secret_a_secret_s_secret_h_secret___secret_u_secret_s_secret_._secret_j_secret_s_secret_o_secret_n_secret_"_secret_,_secret_
+_secret_ _secret_ _secret_ _secret_ _secret_ _secret_ _secret_"_secret_t_secret_r_secret_a_secret_n_secret_s_secret_l_secret_a_secret_t_secret_i_secret_o_secret_n_secret_"_secret_:_secret_ _secret_"_secret_/_secret_m_secret_s_secret_g_secret___secret_h_secret_a_secret_s_secret_h_secret___secret_%_secret_t_secret_w_secret_o_secret___secret_l_secret_e_secret_t_secret_t_secret_e_secret_r_secret_s_secret___secret_c_secret_o_secret_d_secret_e_secret_%_secret_._secret_j_secret_s_secret_o_secret_n_secret_"_secret_,_secret_
+_secret_ _secret_ _secret_ _secret_ _secret_}_secret_,_secret_
+_secret_ _secret_ _secret_ _secret_ _secret_{_secret_
+_secret_ _secret_ _secret_ _secret_ _secret_ _secret_ _secret_"_secret_s_secret_o_secret_u_secret_r_secret_c_secret_e_secret_"_secret_:_secret_ _secret_"_secret_/_secret_s_secret_t_secret_e_secret_a_secret_m_secret___secret_u_secret_s_secret_._secret_j_secret_s_secret_o_secret_n_secret_"_secret_,_secret_
+_secret_ _secret_ _secret_ _secret_ _secret_ _secret_ _secret_"_secret_t_secret_r_secret_a_secret_n_secret_s_secret_l_secret_a_secret_t_secret_i_secret_o_secret_n_secret_"_secret_:_secret_ _secret_"_secret_/_secret_s_secret_t_secret_e_secret_a_secret_m_secret___secret_%_secret_t_secret_w_secret_o_secret___secret_l_secret_e_secret_t_secret_t_secret_e_secret_r_secret_s_secret___secret_c_secret_o_secret_d_secret_e_secret_%_secret_._secret_j_secret_s_secret_o_secret_n_secret_"_secret_,_secret_
+_secret_ _secret_ _secret_ _secret_ _secret_}_secret_,_secret_
+_secret_ _secret_ _secret_ _secret_ _secret_{_secret_
+_secret_ _secret_ _secret_ _secret_ _secret_ _secret_ _secret_"_secret_s_secret_o_secret_u_secret_r_secret_c_secret_e_secret_"_secret_:_secret_ _secret_"_secret_/_secret_g_secret_o_secret_o_secret_g_secret_l_secret_e_secret_p_secret_l_secret_a_secret_y_secret___secret_u_secret_s_secret_._secret_j_secret_s_secret_o_secret_n_secret_"_secret_,_secret_
+_secret_ _secret_ _secret_ _secret_ _secret_ _secret_ _secret_"_secret_t_secret_r_secret_a_secret_n_secret_s_secret_l_secret_a_secret_t_secret_i_secret_o_secret_n_secret_"_secret_:_secret_ _secret_"_secret_/_secret_g_secret_o_secret_o_secret_g_secret_l_secret_e_secret_p_secret_l_secret_a_secret_y_secret___secret_%_secret_t_secret_w_secret_o_secret___secret_l_secret_e_secret_t_secret_t_secret_e_secret_r_secret_s_secret___secret_c_secret_o_secret_d_secret_e_secret_%_secret_._secret_j_secret_s_secret_o_secret_n_secret_"_secret_,_secret_
+_secret_ _secret_ _secret_ _secret_ _secret_}_secret_,_secret_
+_secret_ _secret_ _secret_]_secret_
+_secret_
```

Comment:
What on earth happened here? From my understanding you should have no reason to touch `crowdin.yaml`; or anything in the `intl` folder, really.
Please revert these changes.

Reply:
Also: on GitHub, go to your repo's `Actions` and disable the Crowdin related workflows.

Reply:
I have no idea, never touched crowdin.yaml.... what ill do it delete it push and merge conflicts